VideoLocker/AndroidManifest.xml

```diff
@@ -217,6 +226,37 @@
         <service android:name="org.edx.mobile.services.DownloadSpeedService" />
         <service android:name="org.edx.mobile.services.FetchCourseFriendsService"/>
 
+
+        <!--   Parse Setup -->
+        <!--  https://www.parse.com/apps/quickstart#parse_push/android/existing  -->
+        <service android:name="com.parse.PushService" />
```

Comment:
@aleffert  it maybe hard. I did some research. Looks if we want the white label at this level, maybe we also need to generate manifest file during build time?